WSL/interop.md

```diff
@@ -3,123 +3,183 @@ title: Windows interoperability with Linux
 
-The Windows binary must include the file extension, match the file case, and be executable.  Non-executables including batch scripts and command like `dir` can be run with `/mnt/c/Windows/System32/cmd.exe /C` command. Examples:
+Windows binaries must include the file extension, match the file case, and be executable.  Non-executables including batch scripts.  CMD native commands like `dir` can be run with `cmd.exe /C` command.
+
+Examples:
 
 ``` BASH
-$ /mnt/c/Windows/System32/cmd.exe /C dir
-$ /mnt/c/Windows/System32/PING.EXE www.microsoft.com
+$ cmd.exe /C dir
+<- contents of C:\ ->
+
+$ PING.EXE www.microsoft.com
+Pinging e1863.dspb.akamaiedge.net [2600:1409:a:5a2::747] with 32 bytes of data:
+Reply from 2600:1409:a:5a2::747: time=2ms
 ```
 
-Parameters are passed to the Windows binary unmodified.  
+Parameters are passed to the Windows binary unmodified.
 
 As an example, the following commands will open `C:\temp\foo.txt` in `notepad.exe`:
 
 ``` BASH
-$notepad.exe “C:\temp\foo.txt”
-$notepad.exe C:\\temp\\foo.txt
+$ notepad.exe “C:\temp\foo.txt”
+$ notepad.exe C:\\temp\\foo.txt
 ```
 
-Modifying files located on VolFs (files not under `/mnt/<x>`) with a Windows application is not supported.  By default, WSL attempts to keep the working directory of the Windows binary as the current WSL directory, but will fall back on the instance creation directory if the working directory is on VolFs.
+Modifying files located on VolFs (files not under `/mnt/<x>`) with a Windows application in WSL is not supported.
 
-As an example; `bash.exe` is initially launched from `C:\temp` and the current WSL directory is changed to the user’s home.  When `notepad.exe` is called from the user’s home directory, WSL automatically reverts to `C:\temp` as the notepad.exe working directory:
+By default, WSL tries to keep the working directory of the Windows binary as the current WSL directory, but will fall back on the instance creation directory if the working directory is on VolFs.
+
+As an example; `wsl.exe` is initially launched from `C:\temp` and the current WSL directory is changed to the user’s home.  When `notepad.exe` is called from the user’s home directory, WSL automatically reverts to `C:\temp` as the notepad.exe working directory:
 
 ``` BASH
-C:\temp>bash
+C:\temp> wsl
 /mnt/c/temp/$ cd ~
 ~$ notepad.exe foo.txt
 ~$ ls | grep foo.txt
 ~$ exit
+
 exit
 C:\temp>dir | findstr foo.txt
 09/27/2016  02:15 PM                14 foo.txt
 ```
 
-Users may disable the ability to run Windows binaries for a single WLS session by running the following command as root:  
+## Share environment variables between Windows and WSL
+
+> Available in Windows Insider builds 17063 and later.
+
+Prior to 17063, only Windows environment variable that WSL could access was `PATH` (so you could launch Win32 executables from under WSL).
+
+Starting in 17063, WSL and Windows share `WSLPATH`, a special environment variable created to bridge Windows and Linux distros running on WSL.
```

Comment:
This needs to be changed to 'WSLENV' @scooley

Reply:
Fixed!  Good catch.